docs/security/access-control.md

```diff
@@ -0,0 +1,646 @@
+---
+title: "Access Control"
+slug: /security/access-control
+keyword: security
+license: "This software is licensed under the Apache License version 2."
+---
+
+## Overview
+
+Gravitino adopts RBAC and DAC. 
+
+Role-based Access Control (RBAC): Access privileges are assigned to roles, which are in turn assigned to users or groups.
+
+Discretionary Access Control(DAC): Each metadata object has an owner, who can in turn grant access to that object.
+
+:::info
+
+Gravitino only supports authorization for secureable objects, when it comes to authentication.
+Gravitino doesn't support metadata authentication.
+
+:::
+
+
+## Concept
+
+### Role
+
+A metadata object to which privileges can be granted. Roles are in turn assigned to users or groups.
+
+### Privilege
+
+A defined level of access to an object. Multiple distinct privileges may be used to control the granularity of access granted.
+
+### User
+
+A user identity recognized by Gravitino.  External user system instead of Gravitino manages users. 
+
+### Group
+
+A group identity recognized by Gravitino. External user system instead of Gravitino manages groups. 
+
+### Metadata objects
+
+Metadata objects are managed in Gravitino, such as `CATALOG`, `SCHEMA`, `TABLE`,
+`COLUMN`, `FILESET`, `TOPIC`, `COLUMN`, `ROLE`, `METALAKE`. A metadata object is combined by a `type` and a
+comma-separated `name`. For example, a `CATAGLOG` object has a name "catalog1" with type
+"CATALOG", a `SCHEMA` object has a name "catalog1.schema1" with type "SCHEMA", a `TABLE`
+object has a name "catalog1.schema1.table1" with type "TABLE". `METALAKE` object has a name "metalake1".
+
+### Securable objects
+
+A metadata object to which access can be granted. Unless allowed by a grant, access is denied. 
+Every securable object resides within a logical container in a hierarchy of containers.
+The top container is the metalake. You can understand that metalake a customer organization.
```

Comment:
You can view metalake as a xxxxx.

The word `understand` is not elegant.

Reply:
I can remove this sentence.